src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3735,6 +3735,11 @@ Copyright (C) Microsoft Corporation. All rights reserved.
           Outputs="$(IntermediateOutputPath)%(Culture)\$(TargetName).resources.dll"
           Condition="'@(_SatelliteAssemblyResourceInputs)' != '' and '$(GenerateSatelliteAssembliesForCore)' != 'true'">
 
+    <PropertyGroup>
+      <_SdkToolsPathMaybeWithx64Architecture>$(TargetFrameworkSDKToolsDirectory)</_SdkToolsPathMaybeWithx64Architecture>
```

Comment:
I'm not a huge fan of this name. Isn't it just the SDK tools path, and that sometimes includes x64? Are we already using _SdkToolsPath? If so, maybe _TargetPlatformSpecificSdkToolsPath?

Reply:
I'd actually rather name it `_AlExeToolPath` and set it the same way I'm currently setting `_SdkToolsPathMaybeWithx64Architecture`. There's precedence for this in `Microsoft.Common.CurrentVersion.Targets` with `WinMdExpToolPath`. The name is better since it implies how specific it is and we don't need to worry about making a general solution for a property that isn't in demand much.

Reply:
Works for me